packages/zoe/src/contracts/simpleExchange.js

```diff
@@ -4,47 +4,48 @@ import makePromise from '@agoric/make-promise';
 import { makeZoeHelpers, defaultAcceptanceMsg } from './helpers/zoeHelpers';
 
 /**
- * The SimpleExchange only accepts limit orders. A limit order is an order with
- * payoutRules that specifies wantAtLeast on one side and offerAtMost on the
- * other:
- * [ { kind: 'wantAtLeast', amount2 }, { kind: 'offerAtMost', amount1 }]
- * [ { kind: 'wantAtLeast', amount1 }, { kind: 'offerAtMost', amount2 }]
+ * The SimpleExchange uses Asset and Price as its Issuer roles. In usage,
+ * they're somewhat symmetrical. Participants will be buying or selling in both
+ * directions.
  *
- * Note that the asset specified as wantAtLeast is treated as the exact amount
- * to be exchanged, while the amount specified as offerAtMost is a limit that
- * may be improved on. This simple exchange does not partially fill orders.
+ * { offer: { 'Price', simoleans(5) }, want: { 'Asset', quatloos(3) } }
+ * { offer: { 'Asset', simoleans(5) }, want: { 'Price', quatloos(3) } }
```

Comment:
Our example here needs to use consistent roleNames for simoleans and quatloos. Non-consistent roleNames would fail in practice. 
```suggestion
 * { offer: { 'Asset', simoleans(5) }, want: { 'Price', quatloos(3) } }
 * { offer: { 'Price', quatloos(8) }, want: { 'Asset', simoleans(3) } }
```

Reply:
Done. Thanks.